be/src/util/starrocks_metrics.cpp

```diff
@@ -56,6 +56,13 @@ StarRocksMetrics::StarRocksMetrics() : _metrics(_s_registry_name) {
 
     REGISTER_STARROCKS_METRIC(memtable_flush_total);
     REGISTER_STARROCKS_METRIC(memtable_flush_duration_us);
+    REGISTER_STARROCKS_METRIC(memtable_flush_io_time_us);
+    REGISTER_STARROCKS_METRIC(memtable_flush_bytes_total);
```

Comment:
```suggestion
    REGISTER_STARROCKS_METRIC(memtable_flush_memory_bytes_total);
```

Reply:
The name`memtable_flush_bytes_total` have been used for a long time, and changing it will break the compatibility. I have thought to add a new alia metric `memtable_flush_memory_bytes_total`, but not sure it's worth it.

Reply:
fixed